model/track.js

```diff
@@ -19,7 +19,7 @@ class Track {
     }
 
     belongsToGenre(aGenre){
-        return this.genres.includes(aGenre);
+        return (this.genres || []).includes(aGenre);
```

Comment:
No debería pasar por como creamos los tracks

Reply:
Pero pasaba porque estaban mal puestas las condiciones del handler de consola(permitia el undefined y cuando le iba a hacer no se que metodo, me decia que no le podia pedir eso a undefined). Por las dudas la deje, pero deberia salir siempre por el this.genres

Reply:
:scream: 
Genial entonces